services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/models/SimpleSwitchRule.java

```diff
@@ -0,0 +1,49 @@
+/* Copyright 2019 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.wfm.topology.nbworker.models;
+
+import org.openkilda.model.SwitchId;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class SimpleSwitchRule {
+    private SwitchId switchId;
+    private long cookie;
+    private int inPort;
+    private int outPort;
+    private int inVlan;
+    private int outVlan;
+    private Long meterId;
+    private long pktCount;
+    private long byteCount;
+    private String version;
+
+    @Override
+    public String toString() {
```

Comment:
Since we use lombok, you can define toString using that library and exclude some fields

Reply:
That view is used in response. Therefore, I did not change this method.